### DISCUSSION TITLE
".sprite" File not Working!

### DISCUSSION THREAD

**Discussion by Z-Man01** [SuperTux/supertux#3114] [Q&A] [answered]
What is wrong with the .sprite? I don't see anything wrong in line 100.

![Screenshot from 2024-11-27 20-05-02](https://github.com/user-attachments/assets/3802a275-f5b2-4a88-b4a3-665a92348c40)
![Screenshot from 2024-11-27 20-04-37](https://github.com/user-attachments/assets/fbc2173a-165e-4ecf-8f00-98212b1741a2)

**Comment by provigz** [comment]
Send the full sprite file.

**Reply by Z-Man01** [reply]
The sprite file was been changed a little but still doesn't work.
Here's the zip of the sprite file and the images it's using:
[noloks_ship.zip](https://github.com/user-attachments/files/17949556/noloks_ship.zip)

**Reply by Z-Man01** [reply]
The changed sprite file now shows this:
![Screenshot from 2024-11-28 10-42-09](https://github.com/user-attachments/assets/76fa5662-e672-46f0-83ad-78b29579ce81)

**Reply by Z-Man01** [reply, marked as answer]
I managed to solve the problem. I simply forgot to add these to the image data in the .sprite: "

Before:
![Screenshot from 2024-11-29 09-46-07](https://github.com/user-attachments/assets/0cc7828e-b016-4e06-a4a8-19a4730cc064)


After:
![Screenshot from 2024-11-29 09-46-28](https://github.com/user-attachments/assets/24f999ad-f4f5-4430-a821-40cc86a023bb)
